schema/Retail/RET/on_search.ts

```diff
@@ -391,6 +486,8 @@ export const onSearchSchema = {
                             },
                             code: {
                               type: 'string',
+                              pattern: '^(\\d{8}|\\d{12}|\\d{13}|\\d{14})$',
```

Comment:
it should be in the format code:value
where as per the code, the value can be EAN, HSN, GTIN or others

Need to use regex to segregate and validate the code and value according